common/fw-update-helper.cpp

```diff
@@ -41,57 +45,65 @@ namespace rs2
         RS2_FWU_STATE_FAILED = 3,
     };
 
-    bool is_recommended_fw_available(std::string id)
+    bool is_recommended_fw_available(const std::string& id, const std::string& PID)
```

Comment:
What's `id`?

Reply:
It's the `product_line` string? Why not name it that, then?

Reply:
done